src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -563,13 +563,27 @@ private void MakeConstantTuple(LocalSymbol inProgress, BoundExpression boundInit
                     var type = this.Type;
                     if (boundInitValue == null)
                     {
-                        var inProgressBinder = new LocalInProgressBinder(this, this._initializerBinder);
-                        boundInitValue = inProgressBinder.BindVariableOrAutoPropInitializerValue(_initializer, this.RefKind, type, diagnostics);
+                        recordLocalInBinderChain(this, _initializer, this._initializerBinder);
```

Comment:
>recordLocalInBinderChain(this, _initializer, this._initializerBinder) [](http://example.com/codeflow?start=24&length=69)

It feels like this can be done in ```LocalWithInitializer```'s constructor